_pytest/utilities.py

```diff
@@ -18,7 +18,7 @@ def base_test_conf(pipeline_template):
     return RasaNLUConfig(cmdline_args={
         'response_log': temp_log_file_dir(),
         'port': 5022,
-        "pipeline": registry.registered_pipeline_templates[pipeline_template],
+        "pipeline": registry.registered_pipeline_templates.get(pipeline_template),
```

Comment:
I think optimally this should be `.get(pipeline_template, [])`. Otherwise it returns `None`, which then also overwrites the empty list from the standard config